src/components/itemContextMenu.js

```diff
@@ -99,7 +99,7 @@ import toast from './toast/toast';
             });
         }
 
-        if (!restrictOptions) {
+        if (options.EnableCollectionManagement && !restrictOptions) {
```

Comment:
Should this be requiring admin AND the new permission?

Also should the permission apply to playlists also? or only collections?

Reply:
Permission should only apply to collections and not additionally require admin.

Reply:
This `if` statement is wrapping the options to add to collections and playlists and the `restrictOptions` boolean includes an admin check. (I would also guess that we should just be checking for `browser.tv` instead of only checking specific TV OSes for this.)